test/e2e/infra/secondary_network/secondary_network_prerequisite.sh

```diff
@@ -0,0 +1,360 @@
+export URL_SRIOV_DP_DAEMONSET='https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/master/deployments/k8s-v1.16/sriovdp-daemonset.yaml'
+export URL_WHEREABOUTS_IP_POOLS='https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_ippools.yaml'
+export URL_WHEREABOUTS_OVERLAPPING_IP_RANGE_RES='https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml'
+export YAML_CONFIG_MAP='configMap.yaml'
+export YAML_SRIOV_DP_DAEMONSET='sriovdp-daemonset.yaml'
+export YAML_ANTREA='antrea.yml'
+export YAML_WHEREABOUTS_IP_POOL='whereabouts.cni.cncf.io_ippools.yaml'
+export YAML_WHEREABOUTS_OVERLAPPING_IP_RANGE_RES='whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml'
+export YAML_NET_ATTACH_DEF_CRD='network-attachment-definition-crd.yaml'
+export YAML_SEC_NET_CONFIG_FILE='secondary_network_configuration.yaml'
+export YAML_VIRTUAL_NET_INSTANCE_01='virtual-network-instance1.yaml'
+export YAML_VIRTUAL_NET_INSTANCE_02='virtual-network-instance2.yaml'
+export YAML_VIRTUAL_NET_INSTANCE_03='virtual-network-instance3.yaml'
+#SEC_NET_LOCAL_CONFIG_DIR_PATH is at the test execution server where the required config files are downloaded/generated.
+export SEC_NET_LOCAL_CONFIG_DIR_PATH=$THIS_DIR/secondary_network_config/
+#SEC_NET_REMOTE_CONFIG_DIR_PATH is at the K8s cluster's control node where we configure the cluster for secondary network.
+export SEC_NET_REMOTE_CONFIG_DIR_PATH='/tmp/secondary_network_config/'
+export NAMESPACE='kube-system'
+
+interface=''
+vfs_num=''
+action=''
+configure=''
+
+function validateOptions() {
+	if [ "$interface" != '' ] && [ -z $vfs_num ]; then
+		echoerr "Base interface provided without SRIOV VF counts. Configuration failed."
+		exit 1
+	elif [ -z $interface ] && [ "$vfs_num" != '' ]; then
+		echoerr "SRIOV VF counts provided without base interface. Configuration failed."
+		exit 1
+	fi
+	ssh $hostIP <<EOF
+            export interface=$interface
+            $(typeset -f checkInterface)
+            checkInterface
+EOF
+	if [ $? -eq 255 ]; then
+		echoerr "SSH to $hostIP failed. Ensure ssh configuration needs are met"
+		print_ssh_config_info
+		exit 1
+	elif [ $? -eq 0 ]; then
+		echoerr "Interface doesn't exist at the cluster control node. Configuration failed."
+		exit 1
+	fi
+}
+
+#This function is not required to be called if the --interface-name and --sriov-vf-count params are not provided.
+function checkInterface() {
+	if [[ ! -d /sys/class/net/$interface ]]; then
+		echo "ERROR: No interface named $interface exist on the machine, please check the interface name."
+		exit 1
+	fi
+}
+#This function is not required to be called if the --interface-name and --sriov-vf-count params are not provided.
+function setIPLinkDown() {
+	checkInterface
+	echo "Setting IP Link Down.."
+	ip link set $interface down
+	if [ $? -ne 0 ]; then
+		echo "Failed to set Ip link down!"
```

Comment:
you are not using `echo` / `echoerr` consistently

Reply:
made it consistent (still using echo for info needs. just to avoid it to remote as err).